<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des patients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0,0,127);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion des patients</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>771</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0,0,127);</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter un patient</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>591</width>
          <height>461</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0,0,127);</string>
        </property>
        <property name="title">
         <string>Ajouter un dossier</string>
        </property>
        <widget class="QLineEdit" name="nom">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>100</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>100</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>200</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>dernier rdv</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="drdv">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>200</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>260</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>maladie</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="maladie">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>260</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="obs">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>310</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>310</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>observation</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>150</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="prenom">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>150</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Numero de dossier</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nump">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>60</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ajouter">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>380</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="pic_1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>771</width>
           <height>521</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>pic_1</zorder>
        <zorder>nom</zorder>
        <zorder>label_16</zorder>
        <zorder>label_18</zorder>
        <zorder>drdv</zorder>
        <zorder>label_19</zorder>
        <zorder>maladie</zorder>
        <zorder>obs</zorder>
        <zorder>label_20</zorder>
        <zorder>label_21</zorder>
        <zorder>prenom</zorder>
        <zorder>label_22</zorder>
        <zorder>nump</zorder>
        <zorder>pb_ajouter</zorder>
       </widget>
       <widget class="QLabel" name="p1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p1</zorder>
       <zorder>groupBox_2</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher les patients</string>
       </attribute>
       <widget class="QTableView" name="tabpatient">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>731</width>
          <height>491</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="p2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p2</zorder>
       <zorder>tabpatient</zorder>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Supprimer un patient</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>90</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero du dossier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="p3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p3</zorder>
       <zorder>lineEdit_id_2</zorder>
       <zorder>label_4</zorder>
       <zorder>pb_supprimer</zorder>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Modifier un patient</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>591</width>
          <height>481</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifier un dossier</string>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>70</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>numero de dossier</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m1">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>70</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>240</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>dernier rdv</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m4">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>240</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>290</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>maladie</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m5">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>290</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="m6">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>340</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>340</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>observation</string>
         </property>
        </widget>
        <widget class="QPushButton" name="modif">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>420</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Confirmer</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m3">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>190</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_29">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>140</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_28">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>190</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom </string>
         </property>
        </widget>
        <widget class="QLineEdit" name="m2">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>140</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="p4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <zorder>p4</zorder>
       <zorder>groupBox</zorder>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Rechercher un patient</string>
       </attribute>
       <widget class="QTableView" name="tabr">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>741</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero du dossier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rech">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="rechercher">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>140</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Confirmer</string>
        </property>
       </widget>
       <widget class="QLabel" name="p5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p5</zorder>
       <zorder>tabr</zorder>
       <zorder>label_5</zorder>
       <zorder>rech</zorder>
       <zorder>rechercher</zorder>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Rechercher un ordonnance</string>
       </attribute>
       <widget class="QLineEdit" name="rech2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="rechercher_2">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>150</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.631, y1:0.420455, x2:1, y2:0, stop:1 rgba(0, 0, 127, 255));
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Confirmer</string>
        </property>
       </widget>
       <widget class="QTableView" name="tabr2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>741</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>150</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero du dossier</string>
        </property>
       </widget>
       <widget class="QLabel" name="p6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>531</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>p6</zorder>
       <zorder>rech2</zorder>
       <zorder>rechercher_2</zorder>
       <zorder>tabr2</zorder>
       <zorder>label_7</zorder>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tabWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
